hyph-utf8/tex/generic/hyph-utf8/loadhyph/loadhyph-mk.tex

```diff
@@ -27,7 +27,9 @@
     \input hyph-mk.tex
 \else
     % 8-bit engine (such as TeX or pdfTeX)
-    \message{No Macedonian hyphenation patterns - only for Unicode engines}
+    \message{UTF-8 Macedonian hyphenation patterns}
```

Comment:
This looks semi-OK, but this is a generated file and a complete solution requires changing the sources (written in ruby), so that this file gets generated properly.

After changing the sources a bunch of other files would be created or modified as well (there's a plain text copy of the header, plain text copy of the patterns, there's a file with funny encoding of 8-bit patterns for pTeX, ...)

Reply:
I was not aware they were code-generated. (I noticed there are .tex, .txt with hyphen patters plus other files but since pTex was not enabled (the last branch of if in `loadhyph-mk.tex`), they were not used in Macedonian anyway, so I didn't dare to change something there.) 

I am also OK with your proposal, sounds reasonable, but you probably know better where to edit in this case.

Reply:
Shall we go with accepting this (temporary) solution since changing the ruby files is taking a while?